src/watchmaker/static/config.yaml

```diff
@@ -34,12 +35,14 @@ linux:
         #SaltEL7:
         - dist:
             - redhat
+            - rhel
             - centos
           el_version: 7
           url: https://watchmaker.cloudarmor.io/yum.defs/saltstack/salt/3004.2/salt-reposync-el7-python3.repo
         #SaltEL8:
         - dist:
             - redhat
+            - rhel
```

Comment:
Hmm, this doesn't quite map out correctly... this config was matching on `redhat`, but `SUPPORTED_DISTS` was matching `red hat` (with a space). That implies to me that we're missing something about how the config is mapped to the dist. But I haven't been in this area of the code in a long time, so I can't say exactly what we're missing without tracing things more thoroughly.

Reply:
/shrug

Reply:
Within the code, itself, the explicit dist-references are pretty limited:

~~~
$ find . -type f -print0 | xargs -0 grep -E "red(| )hat"
./src/watchmaker/workers/yum.py:    SUPPORTED_DISTS = ('amazon', 'centos', 'red hat', 'rhel')
./src/watchmaker/static/config.yaml:            - redhat
./src/watchmaker/static/config.yaml:            - redhat
./src/watchmaker/static/config.yaml:            - redhat
./docs/configuration.md:        redhat, centos, and amazon.)
./docs/configuration.md:          - redhat
./docs/configuration.md:            - redhat
./docs/configuration.md:            - redhat
~~~

So, wouldn't really know where to look.

Reply:
Ah. When I'd clicked on the link, the page that rendered was hard to follow. May have been a rendering-blip. 🤷

Reply:
Alright, ea9cf8c appears to address this (tested with a `config.yaml` that omits the `rhel` key)

Reply:
Oh... need to undo this change. 😛